components/ProcessImage.js

```diff
@@ -69,7 +69,7 @@ export default async function ProcessImage( originalImage, setProcessedImage, se
         /*
             Hello world is a c++ function that is linked to react native, it is
             currently being used for OpenCV to count the number of blossoms
-            because Josh cannot figure out how to change the function's name.
+            because we cannot figure out how to change the function's name.
```

Comment:
Hahaha! Could have left that as Josh, I'm not even going to try to figure that one out.

Reply:
:)